src/Tools.md

```diff
@@ -248,6 +253,10 @@ On Linux, the device is likely to appear in the `/dev` directory under a name th
 $ ls /dev/ttyACM*
 ```
 
+> **_Note for VM users_**
+> `ttyACM\*` cannot be found when Pololu is in BOOTSEL mode. To find it, press `RESET`, choose the USB device (in VirtualBox: `Device` -> `USB` -> `Raspberry Pi RP2 Boot`), and you should be able to see it under `/dev`.
```

Comment:
I think this note applies to all users, not just VM users.  In BOOTSEL mode, the device also does not appear for me on my Mac.

Reply:
In that case, do we want to change `Note for VM users:` to `Note:`?

Reply:
@perry0513, I think your comment is wrt https://github.com/lf-lang/embedded-lab/pull/14#discussion_r1302729744? I say we don't need to add colons...

With respect to @edwardalee's comment:

> I think this note applies to all users, not just VM users. In BOOTSEL mode, the device also does not appear for me on my Mac.

In `BOOTSEL` mode, it does appear on _my_ machine. The note is specific to the VM users because it instructs them to add a device filter. That being said, I think the instructions can be clearer.

Reply:
@lhstrh yes I'm referring to that comment